src/libcollections/slice.rs

```diff
@@ -343,15 +343,22 @@ impl<T> [T] {
         core_slice::SliceExt::last_mut(self)
     }
 
-    /// Returns the element of a slice at the given index, or `None` if the
-    /// index is out of bounds.
+    /// Returns a reference to an element or subslice depending on the type of
+    /// index.
+    ///
+    /// - If given a position, returns a reference to the element at that
+    ///   position or `None` if out of bounds.
```

Comment:
Huh? There are more instances of ``None`` without a link in the file. Does it have to be linked every time?